app-games/src/main/java/cm/aptoide/pt/app_games/home/BundlesView.kt

```diff
@@ -25,7 +166,7 @@ fun BundlesView(
       .fillMaxSize()
       .wrapContentSize(Alignment.TopCenter)
   ) {
-    if (viewState.type == LOADING) {
+    if (viewState.type == BundlesViewUiStateType.LOADING) {
       CircularProgressIndicator()
     } else {
       LazyColumn(
```

Comment:
Remove the start padding. Each item in the lazy column already handles its start padding